pkg/cloudprovider/gcp/object_store.go

```diff
@@ -35,10 +35,25 @@ import (
 
 const credentialsEnvVar = "GOOGLE_APPLICATION_CREDENTIALS"
 
+// Interface retriever wraps the GCP SDK functions for accessing object store so they can be faked for testing.
```

Comment:
Godoc comments need to start with what they're describing (`retriever` in this case)

Reply:
I would probably call this bucketWriter for now, and eventually rename it to bucketAccessor once we migrate the rest of this file to the interface. WDYT?

Reply:
bucketWriter works, especially given it's current contents. I'll update.